server/routes/courtRegister/courtRegisterController.test.ts

```diff
@@ -0,0 +1,40 @@
+import type { Request } from 'express'
+import CourtRegisterService from '../../services/courtRegisterService'
+import CourtRegisterController from './courtRegisterController'
+
+jest.mock('../../services/courtRegisterService')
+
+describe('Court Register controller', () => {
+  let courtRegisterService: jest.Mocked<CourtRegisterService>
+  let controller: CourtRegisterController
+  const request = {} as Request
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
is there a better way?

Reply:
find it pretty painful to test these in mock tests. 

We do this in BVL:   
```
const res = ({
    locals: {},
    render: jest.fn(),
    redirect: jest.fn(),
  } as unknown) as jest.Mocked<Response>
```

Would probably just use superagent going forward